iothealth/device_health.py

```diff
@@ -64,7 +64,7 @@ def capacity(cls) -> Dict:
 
     # Override
     @classmethod
-    def temperature(cls) -> Optional[float]:
+    def temperature(cls) -> dict:
```

Comment:
Annotation with `dict` only works on Python 3.9 or newer (https://www.python.org/dev/peps/pep-0585/)

Reply:
I have changed all dict to Dict. Thanks!